tools/installers/install_transformers.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+
+if [ $# != 0 ]; then
+    echo "Usage: $0"
+    exit 1;
+fi
+
+# Install Hugging Face Transformers
+if [ ! -e transformers.done ]; then
+    (
+        set -euo pipefail
+        python3 -m pip install transformers
```

Comment:
Maybe it is better to put the version info of transformers here. What do you think?

Reply:
Any fairly recent `transformers` should work, because the code relies on the `AutoModel` class that was introduced about [2 years ago](https://github.com/huggingface/transformers/releases/tag/1.1.0). Although I'll add a comment that it's tested with 4.9.1.

Reply:
Now after adding special handing for some models that relies on different configuration parameters probably it's a good idea. So I added `>4.9.0`.